lib/stimulus_reflex/reflex.rb

```diff
@@ -129,7 +130,13 @@ def params
     @_params ||= ActionController::Parameters.new(request.parameters)
   end
 
-  def dom_id(record_or_class, prefix = nil)
-    "#" + ActionView::RecordIdentifier.dom_id(record_or_class, prefix).to_s
+  def dom_id(record, prefix = nil)
+    return "#" + [record.model_name.plural, prefix].compact.join("_") if record.is_a?(ActiveRecord::Relation)
+    return "#" + ActionView::RecordIdentifier.dom_id(record, prefix).to_s if record.is_a?(ActiveRecord::Base)
+    "#" + [record.to_s, prefix].compact.join("_")
+  end
+
+  def wrap(content, resource)
```

Comment:
Would love a code comment as to why wrap exists. i.e. to satisfy morphdom when working with lists